alpha-lab/tag-aligner/index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Map Gaze into a Reference 3D Coordinate System"
+description: "Map gaze, head pose, and observer position into a 3D coordinate system of your choice using our Tag Aligner tool."
+permalink: /alpha-lab/tag-aligner/
+meta:
+  - name: twitter:card
+    content: summary
+  - name: twitter:image
+    content: "https://i.ytimg.com/vi/nt_zNSBMJWI/maxresdefault.jpg"
+  - name: twitter:player
+    content: "https://www.youtube.com/embed/vEshPxgWs3E"
+  - name: twitter:width
+    content: "1280"
+  - name: twitter:height
+    content: "720"
+  - property: og:image
+    content: "https://i.ytimg.com/vi/nt_zNSBMJWI/maxresdefault.jpg"
+tags: [Neon, Cloud]
+---
+
+<script setup>
+import TagLinks from '@components/TagLinks.vue'
+</script>
+
+# Map Gaze into a Reference 3D Coordinate System
+
+<TagLinks :tags="$frontmatter.tags" />
+
+<Youtube src="FuzhGwN5t8U"/>
+
+::: tip
+Experience your participants' journey and gaze direction in a real-world environment, digitally reimagined, using Tag Aligner! Our new tool combines Neon eye tracking and headpose data with a user-supplied third-party 3D model.
+:::
+
+## Real-world Positions and Rotations
+
+![Comparison of the different ways to project Neon data into various coordinate systems](./coord-sys-comparisons.png)
+
+It's often important to know the position of an observer in an environment, and how they orient their head and gaze while navigating it. For example, to understand when museum-goers pause in thought while viewing artworks, or how engineers move about and gaze at the surrounding environment during mission-critical tasks.
+
+With a digital twin of a real-world environment, you can visualize an observer's trajectory, and how they direct their gaze spatially onto objects within the scene, by mapping Neon’s gaze + head pose into the digital twin.
+
+In this guide, we'll show you how to do this using data from Neon + Reference Image Mapper (RIM) and the digital twin of your choice.
+
```

Comment:
@rennis250, add a note here, something like:

If you're not already familiar with our RIM enrichment, be sure to check out the [RIM documentation]()

Reply:
done